bears/python/PEP8Bear.py

```diff
@@ -41,14 +42,69 @@ def run(self, filename, file,
                    'max_line_length': max_line_length,
                    'indent_size': indent_size}
 
+        errors = self.list_pep8_errors(source=file, options=options)
```

Comment:
I can confirm that. The [documentation](https://pypi.python.org/pypi/autopep8) states that autopep8 fixes some things not reported by `pycodestyle`